modules/drivers/sparksql/project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject metabase/sparksql-driver "1.0.0-SNAPSHOT-1.2.2"
+(defproject metabase/sparksql-driver "1.0.0-SNAPSHOT-1.3.0"
```

Comment:
The pattern for the driver version numbers here is `1.0.0-SNAPSHOT-<dependency-version>` so this should be `1.0.0-SNAPSHOT-2.3.7`. The reason for `1.0.0-SNAPSHOT` is because Metabase is also technically version `1.0.0-SNAPSHOT` in `project.clj`.

The idea here is we can eventually support some sort of admin plugin management page where you can see what versions of things you're using and some sort of "Metabase Extension Store" where you can add drivers or other plugins to your instance at runtime. If you want to change the version to something like `0.37.0-jdbc-2.3.7` that's fine with me too. But then that would get out of sync pretty quickly. If we remembered to update it, we'd know the last time we changed a driver. `2020-10-02-jdbc-2.3.7` maybe? (`<date-of-last-change>-<dep-version>`)? I'm open to thoughts/suggestions @robdaemon @sbelak. We might want to write a lein plugin to automate this stuff. We could just look at the git commit for the revision date or Metabase version (by looking at the tag)

@sbelak, if you change the version number don't forget to change it in `resources/plugin-manifest.yml` as well since that's the version number we actually read when loading plugins. I guess maybe we should automate that too or at least add comments so people remember to update both